vue.config.js

```diff
@@ -119,6 +119,8 @@ if (process.env.BUILD_ENV === 'local' || process.env.BUILD_ENV === undefined) {
 } else if (process.env.BUILD_ENV === 'camptocamp') {
   config.urls = config.urlsConfigurations.prod;
   config.isProduction = true; // explicit prod flag
+  // set a warning if bundle size is too big
+  result.configureWebpack.performance.hints = 'warning';
```

Comment:
@brunobesson is it yet necessary ? I've increased the limits, it should pass like a charm.

Reply:
Nope. Only forgot you had already made the changes. Feel free to restore the correct behaviour!

Reply:
Ok, will do it on next PR